### ISSUE TITLE
spring @Profile annotation does not work for disabling Actions in integration tests

### ISSUE THREAD

**Issue by aklikic** [lightbend/kalix-jvm-sdk#1499]
I set profle `test` on my `Integration Test` class:
```
@RunWith(SpringRunner.class)
@SpringBootTest(classes = Main.class)
@Profile("test")
public class IntegrationTest extends KalixIntegrationTestKitSupport {
```
And set `!test` profile on my `Action` class:
```
@Subscribe.Topic("input")
@Profile("!test")
public class InputTopicSubscriberAction extends Action {
```

I would expected this to disable the `Action` but it did not.

**Comment by aklikic** [comment]
Reproducable: 
https://github.com/aklikic/city-temperature-aggregation-spring/blob/main/src/it/java/com/example/city/IntegrationTest.java

**Comment by aludwiko** [comment]
Fixed in: #1486
